Userland/Utilities/man.cpp

```diff
@@ -22,9 +21,14 @@
 #include <sys/wait.h>
 #include <unistd.h>
 
-static ErrorOr<pid_t> pipe_to_pager(DeprecatedString const& command)
+static ErrorOr<pid_t> pipe_to_pager(String const& command)
```

Comment:
Please prefer `StringView` over `String const&`

Reply:
Note: In this case it's probably fine either way, but `String const&` has some advantages in regards to refcounting that converting to `StringView` doesn't have, as it loses access to backing information.